staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -106,65 +106,65 @@ var (
 		[]string{priorityLevel, flowSchema},
 	)
 	// PriorityLevelExecutionSeatsObserverGenerator creates observers of seats occupied throughout execution for priority levels
-	PriorityLevelExecutionSeatsObserverGenerator = NewSampleAndWaterMarkHistogramsGenerator(clock.RealClock{}, time.Millisecond,
+	PriorityLevelExecutionSeatsObserverGenerator = NewSampleAndCountHistogramsGenerator(clock.RealClock{}, time.Millisecond*10,
 		&compbasemetrics.HistogramOpts{
 			Namespace:      namespace,
 			Subsystem:      subsystem,
 			Name:           "priority_level_seat_count_samples",
-			Help:           "Periodic observations of number of seats occupied for any stage of execution (but only initial stage for WATCHes)",
+			Help:           "Periodic observations of utilization of seats occupied for any stage of execution (but only initial stage for WATCHes)",
 			Buckets:        []float64{0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1},
 			ConstLabels:    map[string]string{phase: "executing"},
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		&compbasemetrics.HistogramOpts{
+		&compbasemetrics.CounterOpts{
 			Namespace:      namespace,
 			Subsystem:      subsystem,
-			Name:           "priority_level_seat_count_watermarks",
-			Help:           "Watermarks of the number of seats occupied for any stage of execution (but only initial stage for WATCHes)",
-			Buckets:        []float64{0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1},
+			Name:           "priority_level_seat_count_band_secs",
```

Comment:
I guess we probably shouldn't be changing the type of the already exposed metric...
We should deprecate the historical one and introduce a new one instead.

@dgrisonnet @logicalhan - for thoughts

Reply:
Yes, your intuition that we should deprecate the old one and introduce the new one is mostly correct, but since you are renaming this metric, you're effectively deleting the old alpha metric and creating a brand new one. Since there is a memory usage issue with the old one, I'm actually okay with this approach, but definitely we should note this in the release notes, since anyone ingesting the old metric is just going to stop receiving data.